src/Converter/HeaderConverter.php

```diff
@@ -77,6 +77,6 @@ private function createAtxHeader($level, $content)
     {
         $prefix = str_repeat('#', $level) . ' ';
 
-        return $prefix . $content . "\n\n";
+        return "\r\n" . $prefix . $content . "\n\n";
```

Comment:
This library uses Linux line endings.  Would you mind changing this to `\n` only?  (If you want Windows line endings instead, simply do a `str_replace` after the conversion process.)